modules/base/rendering/renderableplane.cpp

```diff
@@ -110,6 +136,18 @@ namespace {
         // [[codegen::verbatim(AutoScaleInfo.description)]]
         std::optional<bool> autoScale;
 
+        // [[codegen::verbatim(ScaleByDistanceInfo.description)]]
+        std::optional<bool> scaleByDistance;
+
+        // [[codegen::verbatim(ScaleRatioInfo.description)]]
+        std::optional<float> scaleRatio;
+
+        // [[codegen::verbatim(ScaleByDistanceMaxHeightInfo.description)]]
+        std::optional<float> scaleByDistanceMaxHeight;
+
+        // [[codegen::verbatim(ScaleByDistanceMinHeightInfo.description)]]
+        std::optional<float> scaleByDistanceMinHeight;
```

Comment:
These should be grouped into a struct (and also a property ower in the UI). Se for example how it's done for the size settings in the point cloud renderable: 
https://github.com/OpenSpace/OpenSpace/blob/17279d2b4ad8b1672087c04e0742fbea2e460107/modules/base/rendering/pointcloud/renderablepointcloud.cpp#L434

I'll gladly discuss how to do it, if you want to!  Have some ideas